src/seeders/20230317073843-dummy-user.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
+/**
+ * @param {data} queryInterface .
+ * @returns {data} .
+ */
+export async function up(queryInterface) {
+  return queryInterface.bulkInsert('Dummy', [{
+    names: 'Dummy User',
+    email: 'dummy@email.com',
+    password: 'dummypassword',
+    createdAt: new Date(),
+    updatedAt: new Date()
+  }]);
+}
+/**
+ * @param {data} .
+ * @returns {data} .
+ */
+export async function down() {
+  /**
+   * Add commands to revert seed here.
+   *
+   * Example:
+   * await queryInterface.bulkDelete('People', null, {});
+   */
+}
```

Comment:
undo the change you did on the UP function above. i.e buldDelete the user you seeded